--- Makefile.orig	2011-10-21 20:48:46.000000000 +0200
+++ Makefile	2016-02-24 19:53:36.378427755 +0100
@@ -1,29 +1,32 @@
 INSTALLDIR = /usr/local
-HINWEIS0 = 'Bitte Konfigurationsdatei in /etc mit Changelog abgleichen'
-HINWEIS1 = 'Check the configuration file in /etc with the Changelog'
-HINWEIS2 = 'Comparez le fichier de config. en /etc avec le Changelog'
+CFLAGS=
+BINDIR=
+MANDIR=
+DATADIR=
+SYSCONFDIR=
+
 
 all:
 	if test -f badsect.o; then rm *.o; fi
-	gcc -g -c badsect.c
-	gcc -g -c cputest.c
-	gcc -g -c dvdbackup.c
-	gcc -g -c dvdcell.c
-	gcc -g -c dvdcopy.c
-	gcc -g -c dvdformat.c
-	gcc -g -c dvdinfo.c
-	gcc -g -c dvdtools.c
-	gcc -g -c ifo.c
-	gcc -g -c lxdvdrip.c
-	gcc -g -c mpeg2dec.c
-	gcc -g -c requant.c
-	gcc -g -c streamanalyze.c
-	gcc -g -c systools.c
-	gcc -g -c tcmemcpy.c
-	gcc -g -c vaporize.c
-	gcc -pthread  -g -o lxdvdrip *o -ldvdread -lm
+	gcc -g $(CFLAGS) $(LDFLAGS) -c badsect.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c cputest.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdbackup.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdcell.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdcopy.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdformat.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdinfo.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c dvdtools.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c ifo.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c lxdvdrip.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c mpeg2dec.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c requant.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c streamanalyze.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c systools.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c tcmemcpy.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -c vaporize.c
+	gcc -pthread  -g $(CFLAGS) $(LDFLAGS) -o lxdvdrip *o -ldvdread -lm
 	if test -f badsect.o; then rm *.o; fi
-	gcc -g -lm -o lxac3scan lxac3scan.c
+	gcc -g $(CFLAGS) $(LDFLAGS) -lm -o lxac3scan lxac3scan.c
 	cd vamps && make && cd ..
 	cd dvdbackup && make && cd ..
 	cd requant && make && cd ..
@@ -37,25 +40,17 @@
 	cd mbuffer && make clean && cd ..
 
 install:
-	cp lxdvdrip $(INSTALLDIR)/bin
-	cp lxac3scan $(INSTALLDIR)/bin
-	cp lxdvdrip.1 $(INSTALLDIR)/man/man1
-	cp lxdvdrip.wav $(INSTALLDIR)/share
+	mkdir -p $(INSTALLDIR)/$(BINDIR)
+	mkdir -p $(INSTALLDIR)/$(MANDIR)/man1
+	mkdir -p $(INSTALLDIR)/$(DATADIR)
+	mkdir -p $(INSTALLDIR)/$(SYSCONFDIR)
+	cp lxdvdrip $(INSTALLDIR)/$(BINDIR)
+	cp lxac3scan $(INSTALLDIR)/$(BINDIR)
+	cp lxdvdrip.1 $(INSTALLDIR)/$(MANDIR)/man1
+	cp lxdvdrip.wav $(INSTALLDIR)/$(DATADIR)
+	install -m 644 lxdvdrip.conf $(INSTALLDIR)/$(SYSCONFDIR)
 	cd vamps && make install && cd ..
 	cd dvdbackup && make install && cd ..
 	cd requant && make install && cd ..
 	cd mbuffer && make install && cd ..
-	if test -f /etc/lxdvdrip.conf; then echo $(HINWEIS0); fi;
-	if test -f /etc/lxdvdrip.conf; then echo $(HINWEIS1); fi;
-	if test -f /etc/lxdvdrip.conf; then echo $(HINWEIS2); else cp lxdvdrip.conf /etc; fi;
 
-uninstall:
-	rm $(INSTALLDIR)/bin/lxdvdrip
-	rm $(INSTALLDIR)/bin/lxac3scan
-	rm $(INSTALLDIR)/man/man1/lxdvdrip.1
-	rm $(INSTALLDIR)/share/lxdvdrip.wav
-	rm $(INSTALLDIR)/bin/vamps_lxdvdrip
-	rm $(INSTALLDIR)/bin/play_cell_lxdvdrip
-	rm $(INSTALLDIR)/bin/dvdbackup_lxdvdrip
-	rm $(INSTALLDIR)/bin/requant_lxdvdrip
-	rm $(INSTALLDIR)/bin/mbuffer_lxdvdrip
--- mbuffer/Makefile.orig	2011-10-24 23:46:37.000000000 +0200
+++ mbuffer/Makefile	2016-02-24 20:19:45.507788003 +0100
@@ -9,10 +9,10 @@
 
 buffer: log.c mbuffer.c network.c
 	if test -f log.o; then rm *.o; fi
-	gcc -g -O2 -c log.c -Wno-unused-result
-	gcc -g -O2 -c mbuffer.c -w
-	gcc -g -O2 -c network.c -Wno-unused-result
-	gcc -pthread -O2 -g -o mbuffer_lxdvdrip *o -lrt -lm
+	gcc $(CFLAGS) -c log.c -Wno-unused-result
+	gcc $(CFLAGS) -c mbuffer.c
+	gcc $(CFLAGS) -c network.c
+	gcc -pthread $(CFLAGS) -o mbuffer_lxdvdrip *o $(LDFLAGS) -lm -lrt
 	if test -f log.o; then rm *.o; fi
 
 clean:
