diff -up lxdvdrip/dvdbackup/dvdbackup_lxdvdrip.c.orig lxdvdrip/dvdbackup/dvdbackup_lxdvdrip.c
--- lxdvdrip/dvdbackup/dvdbackup_lxdvdrip.c.orig	2008-08-20 19:54:26.000000000 +0300
+++ lxdvdrip/dvdbackup/dvdbackup_lxdvdrip.c	2008-08-20 19:55:20.000000000 +0300
@@ -41,9 +41,9 @@
 #include <unistd.h>
 #include <string.h>
 #include <limits.h>
-#include <dvdread/dvd_reader.h>
-#include <dvdread/ifo_read.h>
-#include <dvdread/ifo_print.h>
+#include <libdvdread/dvd_reader.h>
+#include <libdvdread/ifo_read.h>
+#include <libdvdread/ifo_print.h>
 
 
 #define MAXNAME 256
diff -up lxdvdrip/dvdinfo.h.orig lxdvdrip/dvdinfo.h
--- lxdvdrip/dvdinfo.h.orig	2008-08-20 19:55:34.000000000 +0300
+++ lxdvdrip/dvdinfo.h	2008-08-20 19:56:10.000000000 +0300
@@ -15,10 +15,10 @@ COMMENTAIRE :
 #ifndef __DVDINFO_H__
 #define __DVDINFO_H__
 
-#	include <dvdread/dvd_reader.h>
-#	include <dvdread/ifo_read.h>
-#	include <dvdread/ifo_print.h>
-#	include <dvdread/nav_read.h>
+#	include <libdvdread/dvd_reader.h>
+#	include <libdvdread/ifo_read.h>
+#	include <libdvdread/ifo_print.h>
+#	include <libdvdread/nav_read.h>
 
 #ifndef DVD_BLOCK_LEN
 #	define DVD_BLOCK_LEN	2048
diff -up lxdvdrip/dvdtools.h.orig lxdvdrip/dvdtools.h
--- lxdvdrip/dvdtools.h.orig	2008-08-20 19:56:23.000000000 +0300
+++ lxdvdrip/dvdtools.h	2008-08-20 20:16:39.000000000 +0300
@@ -15,10 +15,10 @@ COMMENTAIRE :
 #ifndef __DVDTOOLS_H__
 #define __DVDTOOLS_H__
 
-#	include <dvdread/dvd_reader.h>
-#	include <dvdread/ifo_read.h>
-#	include <dvdread/ifo_print.h>
-#	include <dvdread/nav_read.h>
+#	include <libdvdread/dvd_reader.h>
+#	include <libdvdread/ifo_read.h>
+#	include <libdvdread/ifo_print.h>
+#	include <libdvdread/nav_read.h>
 
 #define 	stAudio		0
 #define		stVideo		1
diff -up lxdvdrip/lxdvdrip.c.orig lxdvdrip/lxdvdrip.c
--- lxdvdrip/lxdvdrip.c.orig	2008-08-20 20:16:52.000000000 +0300
+++ lxdvdrip/lxdvdrip.c	2008-08-20 20:17:34.000000000 +0300
@@ -298,9 +298,9 @@ injection_files
 #include <stdlib.h>
 #include <stdint.h>
 
-#include <dvdread/dvd_reader.h>
-#include <dvdread/ifo_read.h>
-#include <dvdread/ifo_print.h>
+#include <libdvdread/dvd_reader.h>
+#include <libdvdread/ifo_read.h>
+#include <libdvdread/ifo_print.h>
 #include <string.h>
 #include <sys/stat.h>
 #include <sys/types.h>
diff -up lxdvdrip/streamanalyze.c.orig lxdvdrip/streamanalyze.c
--- lxdvdrip/streamanalyze.c.orig	2008-08-20 20:17:49.000000000 +0300
+++ lxdvdrip/streamanalyze.c	2008-08-20 20:18:30.000000000 +0300
@@ -21,9 +21,9 @@
 #include <stdint.h>
 #include <stdlib.h>
 #include <getopt.h>
-#include <dvdread/ifo_read.h>
-#include <dvdread/ifo_types.h>
-#include <dvdread/dvd_reader.h>
+#include <libdvdread/ifo_read.h>
+#include <libdvdread/ifo_types.h>
+#include <libdvdread/dvd_reader.h>
 
 #define VERSION        "0.4"
 #define MAXSTREAMS     50
diff -up lxdvdrip/vamps/play_cell_lxdvdrip.c.orig lxdvdrip/vamps/play_cell_lxdvdrip.c
--- lxdvdrip/vamps/play_cell_lxdvdrip.c.orig	2008-08-20 20:18:41.000000000 +0300
+++ lxdvdrip/vamps/play_cell_lxdvdrip.c	2008-08-20 20:19:13.000000000 +0300
@@ -44,8 +44,8 @@
 #include <string.h>
 #include <getopt.h>
 #include <inttypes.h>
-#include <dvdread/ifo_read.h>
-#include <dvdread/nav_read.h>
+#include <libdvdread/ifo_read.h>
+#include <libdvdread/nav_read.h>
 
 
 /* defines */
